rosidl_generator_py/package.xml

```diff
@@ -29,6 +29,16 @@
   <buildtool_export_depend>rosidl_typesupport_c</buildtool_export_depend>
   <buildtool_export_depend>rosidl_typesupport_interface</buildtool_export_depend>
 
+  <!-- BEGIN: Following 'exec_depend's are likely only needed in test in downstream,
+       so ideally "test_export_depend" (similar to 'build_export_depend') are reasonable.
+       With such a feature unexistent yet, they are 'exec_depend'-ed.
+       See https://github.com/ros2/rosidl_python/pull/199 -->
+  <exec_depend>ament_cmake_cppcheck</exec_depend>
+  <exec_depend>ament_cmake_cpplint</exec_depend>
+  <exec_depend>ament_cmake_flake8</exec_depend>
+  <exec_depend>ament_cmake_pep257</exec_depend>
+  <exec_depend>ament_cmake_uncrustify</exec_depend>
+  <!-- END -->  
```

Comment:
```suggestion
  <!-- TODO: The following 'exec_depend's are only needed in downstream tests,
       so ideally a 'test_export_depend' (similar to 'build_export_depend') would be used.
       Since that feature doesn't exist yet, they are 'exec_depend'-ed.
       See https://github.com/ros2/rosidl_python/pull/199 -->
  <exec_depend>ament_cmake_cppcheck</exec_depend>
  <exec_depend>ament_cmake_cpplint</exec_depend>
  <exec_depend>ament_cmake_flake8</exec_depend>
  <exec_depend>ament_cmake_pep257</exec_depend>
  <exec_depend>ament_cmake_uncrustify</exec_depend>

```